caddy.go

```diff
@@ -736,7 +753,8 @@ func startServers(serverList []Server, inst *Instance, restartFds map[string]res
 			if old, ok := restartFds[addr]; ok {
 				// listener
 				if old.listener != nil {
-					file, err := old.listener.File()
+					var file *os.File
+					file, err = old.listener.File()
```

Comment:
Why this change?

Reply:
Because after initial declaration of "err" linter warning:
```declaration of "err" shadows declaration at ./caddy.go:716```
So, I need to use assignment of err instead of declaration.